doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -57,10 +60,10 @@ The default parameters for ``temperature`` and ``convection-radiation`` are show
 * ``type``: type of boundary condition being imposed. At the moment, choices are:
     * ``noflux`` (default) so that there is no heat transfer boundary condition,
     * ``temperature`` (Dirichlet BC), to impose a given temperature ``value`` at the boundary,
-    * ``convection-radiation`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` and emissivity of the boundary :math:`\mathbf{\epsilon}` following Newton's law of cooling (and heating) and Stefan-Boltzmann law of radiation. Note that the expressions for ``h``, ``Tinf`` and ``emissivity`` can be time-dependent, but the current implementation doesn't allow for space dependence (the expressions are evaluated at the origin).
+    * ``convection-radiation`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` and ``emissivity``` of the boundary :math:`\mathbf{\epsilon}` following Newton's law of cooling (and heating) and Stefan-Boltzmann law of radiation. It is also possible to apply a heat flux using ``heat flux`` (:math:`q_0`, Neumann boundary condition). Note that the expressions for ``h``, ``Tinf``, ``emissivity``, and ``heat flux`` can be time-dependent, but the current implementation does not allow for space dependence (the expressions are evaluated at the origin).
```

Comment:
```suggestion
    * ``convection-radiation`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` and ``emissivity``` of the boundary :math:`\mathbf{\epsilon}` following Newton's law of cooling (and heating) and Stefan-Boltzmann law of radiation. It is also possible to apply a temperature-independent heat flux using ``heat flux`` (:math:`q_0`, Neumann boundary condition). Note that the expressions for ``h``, ``Tinf``, ``emissivity``, and ``heat flux`` can be time-dependent, but the current implementation does not allow for space dependence (the expressions are evaluated at the origin).
```